databricks/koalas/indexes.py

```diff
@@ -580,6 +580,33 @@ def unique(self, level=None):
         sdf = self._kdf._sdf.select(self._scol.alias(self._internal.index_columns[0])).distinct()
         return DataFrame(_InternalFrame(sdf=sdf, index_map=self._kdf._internal.index_map)).index
 
+    # TODO: add error parameter
+    def drop(self, labels):
+        """
+        Make new Index with passed list of labels deleted.
+
+        Parameters
+        ----------
+        labels : array-like
+
+        Returns
+        -------
+        dropped : Index
+
+        Examples
+        --------
+        >>> index = ks.Index([1, 2, 3])
+        >>> index
+        Int64Index([1, 2, 3], dtype='int64')
+
+        >>> index.drop([1])
+        Int64Index([2, 3], dtype='int64')
+        """
+        if not isinstance(labels, (tuple, list)):
+            labels = [labels]
+        sdf = self._internal.sdf[~self._internal.index_scols[0].isin(labels)]
```

Comment:
i think maybe you can use below code to cover Index/MultiIndex cases :)

```python
sdf = self._internal.sdf
conds = []
for index_scol, label in zip(self._internal.index_scols, labels):
    conds.append(index_scol == label)

cond = ~reduce(lambda x, y: x & y, conds)
sdf.where(cond)
```

Reply:
Yeah, @RainFung. I think this comment is correct. Can you update please accordingly?

Reply:
ping @RainFung